### ISSUE TITLE
Fix session start-end querying to include coinciding sessions

### ISSUE THREAD

**Issue by mecha** [RebelCode/rcmod-eddbk-rest-api#27]
## Symptoms

Sessions that only partially lie inside the queried `start` and `end` times are not returned by the `sessions` route.

**For instance**: if the query retrieves from the start of July until the end of July, any sessions that start on July and end in August should be served. This is currently not the case.

## How to Reproduce

1. Create a session that starts on one month and ends on another. Ex: 30th Aug - 1st Sept
2. Query the REST API for sessions in August and September
3. Notice how the session created in step 1 is not returned for either queries.

## Possible Cause

The parameter handling is creating a condition that matches sessions that lie entirely in the given `start` and `end` range. This makes sessions that only partially lie in the range not be retrievable.

## Recommended Solution

The REST API needs to provide sessions that coincide with the given `start` and `end` range, not the sessions that lie _inside_ the given range.

In other words, the condition needs to change from:

```
session.start >= qStart && session.end <= qEnd
```

To:

```
session.end > qStart && session.start > qEnd
```

Where `qStart` and `qEnd` are the queried `start` and `end`. The latter condition will match all sessions that end after the queried `start` and all sessions that start before the queried `end`.
